.travis.yml

```diff
@@ -1,25 +1,31 @@
-sudo: required
 language: go
 
+matrix:
+  include:
+    - os: linux
+      env: TEST_SUITE=test-unit
+    - os: linux
+      env: TEST_SUITE=test-bogo
+    - os: linux
+      env: TEST_SUITE=test-interop
+    # Travis does not support Docker on macOS.
+    - os: osx
+      env: TEST_SUITE=test-unit
+  fast_finish: true
+
 services:
   - docker
```

Comment:
how this will work with mac?

Reply:
It is ignored, from https://travis-ci.org/cloudflare/tls-tris/jobs/517919516:
> Services are not supported on osx

I considered adding it to each individual `os: linux` entry, but that would result in duplication which is why I opted to leave it here. I can move it to the matrix if you prefer that?

Reply:
ok, I agree with you - this is correct, don't change